tests/unit/components/details.js

```diff
@@ -9,47 +9,66 @@ describe('Details', function() {
 
   describe('Passing', function() {
     beforeEach(function() {
-      tree = sd.shallowRender(<Details paths={passDetails}/>);
+      tree = sd.shallowRender(<Details details={passDetails}/>);
     });
 
-    it('should render the baseline', function() {
-      expect(tree.findNode('.baseline')).to.not.be.false;
+    it('should render the PassedTest component', function() {
+      expect(tree.findNode('PassedTest')).to.not.be.false;
     });
 
-    it('should not render the diff', function() {
-      expect(tree.findNode('.diff')).to.be.false;
+    it('should pass the baseline to the PassedTest component', function() {
+      expect(tree.findNode('PassedTest').props.paths.baseline).to.be.equal(
```

Comment:
It seems that `tree.findNode('PassedTest').props` (same goes for the FailedTest) gets repeated in almost every test. Can't you move this in the `beforeEach` under a variable (like `passedTestProps`)?